intellij-plugin/src/main/kotlin/com/apollographql/ijplugin/apollodebugserver/ApolloDebugClient.kt

```diff
@@ -65,6 +71,13 @@ class ApolloDebugClient(
       .addHttpInterceptor(LoggingInterceptor { line ->
         logd("ApolloDebugClient HTTP - $line")
       })
+      // Use a dummy NetworkTransport for subscriptions (which are not used), because the default one
+      // references CoroutineDispatcher.limitedParallelism which doesn't exist in certain IDE versions.
+      // (See https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#coroutinesLibraries)
```

Comment:
TIL. Is there a Youtrack to remove this limitation? We're able to workaround for now but it's not great moving forward.

Reply:
I didn't find one, so I opened https://youtrack.jetbrains.com/issue/IJPL-182123